### DISCUSSION TITLE
Paketmanagers

### DISCUSSION THREAD

**Discussion by niStee** [topgrade-rs/topgrade#675] [Ideas]
I was wondering if you had any thoughts on the possibility of adding Topgrade to Winget and Chocolatey as package managers on the Windows platform, in addition to Scoop?

**Comment by SteveLauC** [comment]
You mean packaging Topgrade to Winget and Chocolatey?

This would be cool and make our Windows users easier, so I am willing to do so, though personally, I am not a Windows user so I cannot do this myself.

**Reply by niStee** [reply]
That's what I meant, yes :)

I look into it.

**Reply by niStee** [reply]
[Trenly](https://github.com/Trenly) has released v. 14.0.1 on [winget](https://github.com/microsoft/winget-pkgs/pull/136492). I am considering automating the process via [winget-releaser](https://github.com/marketplace/actions/winget-releaser).

**Comment by SteveLauC** [comment]
Close as completed in #686

**Comment by niStee** [comment]
Choco is now also live:
https://community.chocolatey.org/packages/topgrade
